doc/swagger-api-description/api.yaml

```diff
@@ -445,36 +589,153 @@ definitions:
+  Property:
     description: Describes all file property.
     type: object
     properties:
+      uuid:
+        type: string
+        description: Property UUID.
       property-name:
         type: string
+        description: Property name.
       property-value:
         type: string
+        description: Property value.
 
   PartLocation:
-    description: Describes the sequence number of file part and its location.
+    description: Describes sequence number of file part and its location.
     type: object
     properties:
       uuid:
         type: string
+        description: PartLocation UUID.
       number:
         type: integer
         format: int32
+        description: Sequence number.
       location:
         $ref: "#/definitions/Cloud"
 
+  AllocationModel:
+    description: Describes allocate storage model ot file.
+    type: object
+    properties:
+      uuid:
+        type: string
+        description: Allocation model UUID.
+      modelType:
+        description: Types of allocation model.
+        enum:
+          - UNION
+          - MORROR
+          - HIGH_AVAILABILITY
+      propertySet:
+        type: array
+        items:
+          $ref: "#/definitions/Property"
+
+  Statistics:
+    type: object
+    description: Contain statistic information about cloud.
+    properties:
+      uuid:
+        type: string
+        description: Statistic UUID.
+      cloudSet:
+        $ref: "#/definitions/Cloud"
+      uploadFiles:
+        description: Count of uploaded files.
+        type: integer
+        format: int32
+      uploadTime:
+        description: Time what files was uploaded.
+        type: integer
+        format: int64
+      downloadFiles:
+        description: Count of downloaded files.
+        type: integer
+        format: int32
+      downloadTimes:
+        description: Time what files was downloaded.
+        type: integer
+        format: int64
+
+  HelthStatistis:
+    type: object
+    description: Contain helth-check statistic information about set of clouds.
+    properties:
+      uuid:
+        type: string
+        description: HelthStatistis UUID.
+      helthCloudSet:
+        type: array
+        items:
+          $ref: "#/definitions/HelthCloud"
+
+  HelthCloud:
+    type: object
+    description:  Contain key-value information about cloud and its helth status.
+    properties:
+      cloud:
+        $ref: "#/definitions/Cloud"
+      isAvailable:
+        description: Is cloud available?
+        type: boolean
+
+  CloudSpaseSet:
+    type: object
+    description: Contain set of information about clouds availability space.
+    properties:
+      uuid:
+        type: string
+        description: CloudSpaseSet UUID.
+      cloudSpaceSet:
+        type: array
+        items:
+          $ref: "#/definitions/CloudSpase"
+
+  CloudSpase:
+    type: object
+    description: Contain information about cloud availability space.
+    properties:
+      uuid:
+        type: string
```

Comment:
Do we need a Uuid here? a _cloud_ under _ref_ param already contains it's own Uuid so we know which cloud we are talking about and it's available space.